src/components/Policy.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Card, Accordion, Button, ListGroup } from "react-bootstrap"
+export const Policy = () => {
+
+    return (
+        <Card>
+            <Card.Header>Privacy Policy</Card.Header>
+            <Accordion>
+                <Card>
+                    <Card.Header>
+                        <Accordion.Toggle as={Button} variant="outline-secondary" eventKey="0">
+                        Crypto Pro may use, or disclose the personal information it collects for one or more of the following business purposes:
+                        </Accordion.Toggle>
+                    </Card.Header>
+                        <Accordion.Collapse eventKey="0">
+                            <Card.Body> -To fulfill or meet the reason you provided the information. For example, to provide services.
+-To help maintain the safety, security, and integrity of Crypto Pro’s site and services. For example, Crypto Pro may screen and use the information to protect against spammers, hackers, and others who pose harm to the site and services; Crypto Pro may share information with moderation service providers; and Crypto Pro may share the information with law enforcement.
+-To evaluate and improve the site and services. For example, Crypto Pro may aggregate and/or de-identify information to produce statistical data.
+-To monitor and enforce user compliance with the guidelines of the law and its policies. 
+-To respond to law enforcement requests and as required by applicable law, court order, or governmental regulations.
+</Card.Body>
```

Comment:
This section is not displaying as a list on the hosted site. Is this expected to be displayed as a list?